ironfish/src/rpc/routes/wallet/getTransaction.test.ts

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Assert } from '../../../assert'
+import { useAccountFixture, useTxSpendsFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route wallet/getAccountTransaction', () => {
+  const routeTest = createRouteTest(true)
+
+  it('gets transaction by account', async () => {
+    const node = routeTest.node
+    const account = await useAccountFixture(node.wallet, 'account')
+
+    const { transaction } = await useTxSpendsFixture(node, { account })
+
+    const response = await routeTest.client.wallet.getAccountTransaction({
+      hash: transaction.hash().toString('hex'),
+      account: account.name,
+    })
+
+    expect(response.status).toBe(200)
+
+    const { transaction: responseTransaction, account: responseAccount } = response.content
+
+    Assert.isNotNull(responseTransaction)
+
+    expect(responseAccount).toMatch(account.name)
+
+    expect(responseTransaction.spends).toHaveLength(1)
```

Comment:
Can you compare it with `transaction.spends`, and it would be nice if you can compare the each attributes in spends